nsist/pypi.py

```diff
@@ -181,7 +182,15 @@ def extract_wheel(whl_file, target_dir):
     for p in td.iterdir():
         if p.suffix not in {'.data', '.dist-info'}:
             if p.is_dir():
-                shutil.copytree(str(p), str(target / p.name))
+                try:
+                    shutil.copytree(str(p), str(target / p.name))
+                except OSError as e:
```

Comment:
I'm not entirely sure either, but your example with PyQtCharts is fairly compelling. And if it's going to change anyway...

It might be worth checking that the packages add different files to the shared directory, and throwing an error if there's a conflict at the file level. But that can be in a separate PR.